### ISSUE TITLE
OrientDB 2.1.3: Script command fails

### ISSUE THREAD

**Issue by dennwc** [istreamdata/orientgo#39]
OrientDB 2.1.3 throws this exception, when trying to execute any script command:

```
OrientDB Server Exception: 
  com.orientechnologies.orient.core.exception.OSerializationException:
    Error on unmarshalling content. Class: s
  java.lang.IllegalArgumentException:
    No enum constant com.orientechnologies.orient.core.command.OCommandDistributedReplicateRequest.DISTRIBUTED_EXECUTION_MODE.<script body here>
```

**Comment by dennwc** [comment]
There was an incompatible change in https://github.com/orientechnologies/orientdb/commit/25959255e96705ecf770960103ebf3463a36f54d.
Tracking https://github.com/orientechnologies/orientdb/issues/5085 to get some feedback.

**Comment by dennwc** [comment]
Fixed in OrientDB 2.1.4
